jest.setup.ts

```diff
@@ -1,5 +1,6 @@
 import '@testing-library/jest-dom';
 import fetch from 'jest-fetch-mock';
+import moment from 'moment-timezone';
```

Comment:
Please, consider to use `timezone-mock` instead of `moment` for development environment (it's already used in mono-repository). Then is test:

```
import {
  register as registerTimezone,
  unregister as unregisterTimezone,
} from 'timezone-mock';

beforeAll(() => {
  registerTimezone('UTC');
});

afterAll(() => {
  unregisterTimezone();
});
```